app/assets/javascripts/discourse/app/components/modal/user-status.hbs

```diff
@@ -15,30 +19,31 @@
       <label class="control-label">
         {{i18n "user_status.remove_status"}}
       </label>
+
       <TimeShortcutPicker
         @timeShortcuts={{this.timeShortcuts}}
         @hiddenOptions={{this.hiddenTimeShortcutOptions}}
         @customLabels={{this.customTimeShortcutLabels}}
         @prefilledDatetime={{this.prefilledDateTime}}
-        @onTimeSelected={{action "onTimeSelected"}}
+        @onTimeSelected={{this.onTimeSelected}}
         @_itsatrap={{this._itsatrap}}
       />
     </div>
-    <div class="modal-footer control-group">
```

Comment:
There as some custom styling here, I just tried to rely on default modal-footer behavior instead.